first-party/jib-extension-3147/src/main/java/com/google/cloud/tools/jib/extension/e3147/ClasspathMainClassExtension.java

```diff
@@ -0,0 +1,71 @@
+package com.google.cloud.tools.jib.extension.e3147;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import com.google.cloud.tools.jib.api.buildplan.ContainerBuildPlan;
+import com.google.cloud.tools.jib.gradle.extension.GradleData;
+import com.google.cloud.tools.jib.gradle.extension.JibGradlePluginExtension;
+import com.google.cloud.tools.jib.maven.extension.JibMavenPluginExtension;
+import com.google.cloud.tools.jib.maven.extension.MavenData;
+import com.google.cloud.tools.jib.plugins.extension.ExtensionLogger;
+import com.google.cloud.tools.jib.plugins.extension.ExtensionLogger.LogLevel;
+import com.google.cloud.tools.jib.plugins.extension.JibPluginExtensionException;
+
+public class ClasspathMainClassExtension implements JibGradlePluginExtension<Void>, JibMavenPluginExtension<Void> {
+
+	@Override
```

Comment:
Formatting is wrong, so build will fail. Just run `./gradlew googleJavaFormat` (or `./gradlew goJF` for short) at the root directory (i.e., `jib-extensions/first-party`) to auto-format the code.